osu.Game.Tests/Visual/UserInterface/TestSceneOverlayHeaders.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Containers;
+using osu.Game.Overlays;
+using System;
+using System.Collections.Generic;
+using osu.Framework.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Graphics;
+using osu.Framework.Graphics.Shapes;
+using osuTK.Graphics;
+
+namespace osu.Game.Tests.Visual.UserInterface
+{
+    public class TestSceneOverlayHeaders : OsuTestScene
```

Comment:
Please just call it `TestSceneOverlayHeader` to match conventions (you are testing `OverlayHeader` not `OverlayHeaders`)